frontend/javascripts/admin/dataset/dataset_upload_view.js

```diff
@@ -304,8 +310,19 @@ class DatasetUploadView extends React.Component<PropsWithFormAndRouter, State> {
         this.setState({ isRetrying: false, uploadProgress: resumableUpload.progress() });
       });
 
-      resumableUpload.on("fileRetry", () => {
-        this.setState({ isRetrying: true });
+      resumableUpload.on("fileRetry", file => {
+        const currentRetries = this.state.retries + 1;
+        // log to airbrake in intervals of retriesUntilAnalyticsEventSent
+        if (currentRetries >= retriesUntilAnalyticsEventSent && currentRetries % 10 === 0) {
```

Comment:
I think, the `10` is also worth extracting.

Reply:
See my other comment. When using `_.throttle`, there is only the need for one single variable in my opinion (e.g., report every 60 seconds).